arduino/smartcar/smartcar.ino

```diff
@@ -155,7 +164,68 @@ void loop()
     
 }
 
-//If the connection breaks, this method will be called
```

Comment:
You had removed one of my comments and put your code between my comment and my method. Just be careful when you add code that it doesn't affect other methods/functions.

Reply:
I see that but at the end I did put a new comment with your last well what method